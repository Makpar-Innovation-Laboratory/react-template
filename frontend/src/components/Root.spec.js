import React from "react";
import { shallow } from "enzyme";
import Root from "./Root";

describe("JSX Tests :: <Root /> Rendering", () => {

  test("Sanity Check Snapshot Rendering", () => {
    const props = ({
      store: {
        subscribe: jest.fn(),
        dispatch: jest.fn(),
        getState: jest.fn(),
      },
      history: {}
    });
    const wrapper = shallow(<Root {...props} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
