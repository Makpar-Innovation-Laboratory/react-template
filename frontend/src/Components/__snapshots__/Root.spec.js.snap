// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSX Tests :: <Root /> Rendering Sanity Check Snapshot Rendering 1`] = `
"<Provider store={{...}}>
  <Connect(ConnectedRouterWithContext) history={{...}}>
    <App />
  </Connect(ConnectedRouterWithContext)>
</Provider>"
`;
